src/core/skills/game_mode/pve/pve_jienu.ts

```diff
@@ -48,30 +48,49 @@ export class PveJieNu extends TriggerSkill {
 
   async onEffect(room: Room, event: ServerEventFinder<GameEventIdentifiers.SkillEffectEvent>) {
     const { fromId } = event;
+    let x=0
+    let y=0
     const unknownEvent = event.triggeredOnEvent as ServerEventFinder<GameEventIdentifiers>;
     const identifier = EventPacker.getIdentifier(unknownEvent);
     if (
       identifier === GameEventIdentifiers.DamageEvent ||
       (identifier === GameEventIdentifiers.PhaseStageChangeEvent &&
         room.getPlayerById(fromId).Hp < room.getPlayerById(fromId).LostHp)
     ) {
+      if (identifier === GameEventIdentifiers.DamageEvent) {
+        const damageEvent = unknownEvent as ServerEventFinder<GameEventIdentifiers.DamageEvent>;
+        if (damageEvent.damageType === DamageType.Normal) {
+          x=0; y=0
```

Comment:
prettier this code, and also get avoid using `x` or `y` as variable names, they are meaningless.